docs/content/preview/integrations/camunda.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Camunda
+linkTitle: Camunda
+description: Use Camunda with YSQL 
+aliases:
+section: INTEGRATIONS
+menu:
+  preview:
+    identifier: camunda
+    weight: 571
+isTocNested: true
+showAsideToc: true
+---
+
+[Camunda](https://camunda.com/) is a Java-based framework supporting BPMN for workflow and process automation, CMMN for Case Management and DMN for Business Decision Management. This page shows how it can be integrated with YugabyteDB's YSQL API.
+
+## Prerequisites
+
+Before you start using the KairosDB plugin, ensure that you have:
+Make sure you have the following set of tools installed:
+
+- Java JDK 1.8+,
+- A modern web browser (recent Firefox, Chrome or Microsoft Edge will work fine).
+- Eclipse integrated development environment (IDE)/Visual Studio Code.
+- A YugabyteDB cluster. Refer to [YugabyteDB Quick start guide](/preview/quick-start/) to install and start a local cluster.
+- YugabyteDB JDBC Driver jar [file](https://repo1.maven.org/maven2/com/yugabyte/jdbc-yugabytedb/42.3.5-yb-1/jdbc-yugabytedb-42.3.5-yb-1.jar).
```

Comment:
Things that are only required to be downloaded (and not setup/installation required) can be skipped from pre-req, I think. You'll anyway mention them at appropriate section below.